pipeline_tools/pipelines/optimus/optimus.py

```diff
@@ -55,6 +58,26 @@ def create_optimus_input_tsv(uuid, version, dss_url):
     sample_id = metadata_utils.get_sample_id(primary_bundle)
     print('Writing sample ID to sample_id.txt')
     with open('sample_id.txt', 'w') as f:
-        f.write('{0}'.format(sample_id))
+        f.write(f"{sample_id}")
+
+    species_references = REFERENCES[metadata_utils.get_ncbi_taxon_id(primary_bundle)]
+    print('Writing species references')
+    for key, value in species_references.items():
+        with open(f"{key}.txt", 'w') as f:
+            f.write(f"{value}")
 
     print('Finished writing files')
+
+
+REFERENCES = {
```

Comment:
Minor style nit, but I usually see python "constants" defined at at the top of a file right after the import statements